Dockerfile

```
@@ -3,14 +3,20 @@ ARG BASE_CONTAINER_VERSION=latest
 
 FROM ${REGISTRY}/python3.12-builder:${BASE_CONTAINER_VERSION} as builder
 
-COPY --from=ghcr.io/astral-sh/uv:0.7.12 /uv /uvx /bin/
-
 COPY ./ /aliquotmaf
 
 WORKDIR /aliquotmaf
 
-#RUN pip install tox && tox -e build
-RUN uv build
+ENV PIP_INDEX_URL=https://pypi.org/simple
+ENV PIP_EXTRA_INDEX_URL=https://nexus.osdc.io/repository/pypi-all/simple
+
```

```suggestion
ARG PIP_INDEX_URL
```

This is set during the gitlab build step, no need to hardcode here